src/lib/relationship_graph.cc

```diff
@@ -25,9 +25,9 @@
 #include <boost/graph/connected_components.hpp>
 #include <boost/range/algorithm/find.hpp>
 
-#define DNG_GL_PREFIX "GL-"
-#define DNG_SM_PREFIX "SM-" // define also in newick.cc
-#define DNG_LB_PREFIX "LB-"
+#define DNG_GL_PREFIX "GL/"
+#define DNG_SM_PREFIX "SM/" // define also in newick.cc
+#define DNG_LB_PREFIX "LB/"
```

Comment:
Can we move these to a header file? So we can keep newick.cc in sync with relationship_graph?